client/my-sites/site-settings/placeholder-disconnect.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormattedHeader from 'components/formatted-header';
+import Main from 'components/main';
+
+const Placeholder = ( { translate } ) => {
```

Comment:
Even though we are using default export, it might be nice to provide a more specific name for this component.